README.md

```diff
@@ -29,7 +29,22 @@ TODO
 
 ## CDF chart
 
-TODO
+The CDF chart plots the fraction of the pushes with a duration less than a value vs the durations of all the pushes in the push def. This is visualized as a step plot, such that each step represents a single push. There are vertical lines indicating the 10th, 50th, and 90th percentiles to show to the user where the majority of the data lies. If the visited one push page represents a completed push, a vertical alpha channel appears at the duration value of this push so the user can see how this current push’s duration compares to the durations of all the other completed pushes in the push def. 
```

Comment:
Two suggestions: s/vertical lines/dotted vertical lines/ and s/visited one push page/visited page/.

I think the term 'a vertical alpha channel' is a bit confusing. Maybe 'a vertical translucent line'?